### PR TITLE
Resolve UI bugs for diet goals

### PR DESCRIPTION
**Pull request by yicheng-toh**
Ensure that the tab spaces for the entries are consistent.

### CODE REVIEW

File: src/main/java/athleticli/ui/Parser.java [hunk 1]
```diff
@@ -40,7 +40,7 @@ public class Parser {
     private static final String CALORIES_MARKER = "calories";
     private static final String PROTEIN_MARKER = "protein";
     private static final String CARB_MARKER = "carb";
-    private static final String FAT_MARKER = "fat";
+    private static final String FAT_MARKER = "fats";
```
Comment (DaDevChia) [inline src/main/java/athleticli/ui/Parser.java:43]: One suggestion would be to move the markers to the newly created parameters class.

**Review by DaDevChia — COMMENTED** [review]
LGTM, added suggestion to bring the marker parameters to the newly created parameters class

**Review by DaDevChia — APPROVED** [review]
LGTM, added suggestion to bring the marker parameters to the newly created parameters class.

**Comment by yicheng-toh** [comment]
> LGTM, added suggestion to bring the marker parameters to the newly created parameters class.

Raised issue #38 to address it in the next milestone.

**Pull request merged**